tests/test_agents/test_llama_index_output_handler.py

```diff
@@ -58,17 +63,17 @@ def agent():
 
     except ModuleNotInstalled:
```

Comment:
LlamaIndex should always be installed in test environments, so let's remove this try-except block.

Also, let's replace DuckDuckGoSearchRun with some mock tool in this and other tests, because it's quite possible we'll get rid of this dependency one day.